### PR TITLE
Fix#1069 Ensures that only selected plugins run

### PR DESCRIPTION
**Pull request by Ashrith-Shetty**
On selecting a particular plugin to run, all plugins were automatically getting selected and running,

## Description

1. Sending the data as plugin_data.

_Before the change_
`0=g&1=r&2=o&3=u&4=p&5=%3D&6=w&7=e&8=b&9=%26&10=t&11=y&12=p&13=e&14=%3D&15=s&16=e&17=m&18=i&19=_&20=p&21=a&22=s&23=s&24=i&25=v&26=e&27=%26&28=c&29=o&30=d&31=e&32=%3D&33=O&34=W&35=T&36=F&37=&38=I&39=G&40=&41=0&42=0&43=4&44=%26&45=i&46=d&47=%3D&48=1&49=2&50=%26&51=f&52=o&53=r&54=c&55=e&56=_&57=o&58=v&59=e&60=r&61=w&62=r&63=i&64=t&65=e&66=%3D&67=f&68=a&69=l&70=s&71=e`

_After the change_
`plugin_data: group=web&type=semi_passive&code=OWTF-IG-004&id=4&force_overwrite=false`

2. Convert all self.request.arguments from byte to character (Unicode) string.

_Before the change_
{'plugin_data': [b'group=web&type=semi_passive&code=OWTF-IG-004&id=5&force_overwrite=false']}

_After the change_
{'plugin_data': ['group=web&type=semi_passive&code=OWTF-IG-004&id=5&force_overwrite=false']}

3.  Parse the data in required format by filter_data in post()

_Before the change_
{'plugin_data': ['group=web&type=semi_passive&code=OWTF-IG-004&id=7&force_overwrite=false']

_After the change_
{'group': ['web'], 'type': ['semi_passive'], 'code': ['OWTF-IG-004'], 'id': ['10'], 'force_overwrite': ['false

## Related Issue

https://github.com/owtf/owtf/issues/1069

## Reviewers

@viyatb @sharmamohit123 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [X] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: owtf/api/handlers/work.py [hunk 1]
```diff
@@ -346,7 +361,9 @@ def post(self, work_id=None, action=None):
             if not target_list:
                 raise APIError(400, "Target list should not be empty")
             force_overwrite = str2bool(self.get_argument("force_overwrite", "False"))
```
Comment (sharmamohit123) [inline owtf/api/handlers/work.py]: `force_overwrite` variable needs to be updated after the above change.

Comment (Ashrith-Shetty) [reply owtf/api/handlers/work.py]: Thank you for reviewing the changes.
I will look into it and make the required changes.

Comment (viyatb) [reply owtf/api/handlers/work.py]: correct, thanks @sharmamohit123

Comment (Ashrith-Shetty) [reply owtf/api/handlers/work.py]: @viyatb @sharmamohit123  Have made the required changes, Thank You.

**Comment by sharmamohit123** [comment]
lgtm. ^@viyatb!

**Pull request merged**